go-controller/pkg/libovsdbops/meter.go

```diff
@@ -1,118 +1,63 @@
 package libovsdbops
 
 import (
-	"context"
-	"fmt"
+	"reflect"
 
 	libovsdbclient "github.com/ovn-org/libovsdb/client"
 	"github.com/ovn-org/libovsdb/ovsdb"
+	"k8s.io/apimachinery/pkg/util/sets"
 
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
-	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
 )
 
-// FindMeterByName finds an ovn meter by name, will return `errNotFound` if it does not exist
-func FindMeterByName(nbClient libovsdbclient.Client, name string) (*nbdb.Meter, error) {
-	searched := &nbdb.Meter{
-		Name: name,
-	}
-
-	ctx, cancel := context.WithTimeout(context.Background(), types.OVSDBTimeout)
-	defer cancel()
-	err := nbClient.Get(ctx, searched)
-	if err != nil {
-		return nil, err
-	}
-
-	return searched, nil
+func equalsMeterBand(a, b *nbdb.MeterBand) bool {
```

Comment:
We can't, we don't want to consider the UUID which is the thing we are looking for

Reply:
yes I thought so, but you can't use equalMeterBandExternalIDs instead of reflectDeepEqual as well? I am just wondering since we have the utility why not use it? anyways just a nit, not sure if it actually has huge perf impact since meters are all created just once on startup.

Reply:
Ah, no, that's not exported from the generated `nbdb` package. Certainly we should have a function to compare external IDs and in general to operate with `map[string]string` because we are already using `reflect.DeepEqual` elsewhere. But I think we can cut it out of the scope of this PR.